pyproject.toml

```diff
@@ -0,0 +1,108 @@
+[tool.poetry]
+name = "mentorship-backend"
+version = "1.0.0"
+description = " Backend REST API for the Mentorship System."
+authors = ["Isabel Costa <unknown>"]
+maintainers = ["Isabel Costa <unknown>"]
+license = "GNU GPL v3"
+readme = "README.md"
+homepage = "https://github.com/anitab-org/mentorship-backend"
+repository = "https://github.com/anitab-org/mentorship-backend"
+documentation = "https://github.com/anitab-org/mentorship-backend/wiki"
+keywords = ["REST API",
+    "Anita Borg",
+    "Poetry",
+    "Flask",
+    "Women in Technology"]
```

Comment:
I have a hard time digesting `poetry`'s suitable for just dependency management  when using with Django, because of the way it enforces package information, which is largely redundant here for the most part.

This is why my mind says to go with `pip-tools` when working with Django, even though I myself really like Poetry! :laughing:

Reply:
I don't have much experience with Django so you're probably right😁Idk how the situation looks like when using Flask

Reply:
Oh I mean both Django/Flask, apologies, anything that is not a Python package, basically.